lib/fs-merge-tree.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-const walkSync = require('walk-sync');
-
+var canSymlink = require('can-symlink')();
 const FSTree = require('./');
+var defaultIsEqual = FSTree.defaultIsEqual;
```

Comment:
Seeing a mixture of `var` and `let`, I assume the code that was copied over was all using var.. Shall we update this?